### ISSUE TITLE
Add Case Study page

### ISSUE THREAD

**Issue by selineriz** [aces/gbc_website#15]
@3design:
Case studies: CBrain and LORIS are perfect candidates for this. Infantix from Mitch and team would be another great case study highlighting now neuroscience can be used in clinical practice. Ernesto (cc’d) can provide the materials downstream.

@christinerogers:
Hi @3design thanks for adding this issue.
Could you please add a few more words above to note who/where/when this request came from, e.g. (making this up) from email from F.Mushtaq sept 10 -- so that we have this context handy when time comes to address/process such issues.

Generally let's do this going forward for issues which are requests from different people,
cc @selineriz fyi too

Old repo link: https://github.com/3design/gbc/issues/38

**Comment by 3design** [comment]
@christinerogers That's a good point. I'm searching through my emails and on MS Teams to see if I can track that info down

**Comment by 3design** [comment]
From: Faisal Mushtaq via email on Sept 8, 2024.

Please add a Case Studies page.

Case studies: CBrain and LORIS are perfect candidates for this. Infantix from Mitch and team would be another great case study highlighting how neuroscience can be used in clinical practice. 

Ernest has provided the link to Infantix's Instagram

[https://www.instagram.com/neuronic_infantix?igsh=MTRjMjVqNGNla2pqaA=](https://www.instagram.com/neuronic_infantix?igsh=MTRjMjVqNGNla2pqaA==)

update from Christine: the source is Ernesto Velarde Reyes <velarde@cneuro.edu.cu>

**Comment by 3design** [comment]
![Image](https://github.com/user-attachments/assets/1a18d025-67d1-44b9-9846-f5d4282286b3)

@christinerogers here is a first pass at the case studies page. We will need text for:

- The top section that describes the case studies included in this page
- CBRAIN text is currently the intro text on the website. This can be altered
- LORIS text is also the intro text on the website and might need updating
- Infantix text is a quote I pulled from an article [here](https://en.granma.cu/cuba/2021-06-25/infantix-a-cuban-screening-system-for-our-children) and needs to be replaced

**Comment by christinerogers** [comment]
Agreed the page looks good --- thanks Derek, especially for resourcefully compiling the current content. 

The content will need polishing before it goes live and e.g. EEGNet should be a Case study perhaps even instead of Loris and CBrain for starters. 
Will think about the best way to tackle this and with whom. 

@3design meanwhile Could you please put the page URL here so selin and I can preview it? 

Please Note for next time this should be a PR instead of a direct commit so that it's not live, though because it's not linked from the navigation it's invisible so fine for now.

**Comment by 3design** [comment]
Hi @christinerogers , this hasn't been pushed to main. It's on this [branch](https://github.com/aces/gbc_website/tree/Oct-16-2024-create-case-studies-page) and I haven't created a PR yet as there are many changes still needed.

To preview it, you might have to look at it locally by changing your local repo to that branch code, then running the `npm start` command as described on the [readme under running locally](https://github.com/aces/gbc_website/tree/Oct-16-2024-create-case-studies-page?tab=readme-ov-file#running-locally). Let me know if there's any issues getting this to work on your local drive

**Comment by christinerogers** [comment]
Hi @3design could you please "replace" Loris and CBrain with `EEGNet` and `Harmonized Quantitative EEG Pipeline` ? 

 "replace" i.e. please comment out Loris and Cbrain, don't delete this content, we may use it later. 

* For the EEGNet card, try taking boilerplate and a suitable image from EEGNet.org as a first stab and we'll go from there. 

* For the `Harmonized...` card - please take material from [this slide](https://docs.google.com/presentation/d/1wtXln2rvvo45jHZUEZyF2nSM2HDknwwoSda4Pp36mBE/edit#slide=id.g2fc81fed60c_1_154) including the github address on the slide. For text please also add the Goals section (inclluding bullets) from the [Norms project page](https://globalbrainconsortium.org/project-eegNorms/).  

When you update this PR, please put a preview link / screenshot if you can, then re-assign to me and I"ll have a go refining the text when ready.

**Comment by 3design** [comment]
![Image](https://github.com/user-attachments/assets/adedc2d2-ee22-4dbc-b1e2-67445746cd98)

@christinerogers I've updated the case studies page as requested. For the `harmonized...` I used the text from the Norms project page and a screenshot from the slide. All the links (header, image, and button) on that card go to the github repo

**Comment by christinerogers** [comment]
Hi @3design -- If you're already off for the holidays, just slack me (in january not now) to let me know when you are getting to it. 

When you have a moment could you please merge and publish this?   It must be visible from the menu bar as well. 

If it needs anything from John or Andy to push to production, let me know - thanks.

**Comment by 3design** [comment]
@christinerogers branch has been merged and results are live.
